governance/reviews/template.md

```diff
@@ -23,171 +11,213 @@ Exemplary: project has an extraordinary level of governance development and impl
 Satisfactory: project has appropriate governance for its maturity level and is following that governance
 Mostly Satisfactory: project has mostly appropriate governance, but needs to fix one or two things
 Needs Work: project's governance is lacking and inadequate for its current level of maturity, and needs substantial work to overcome that
-
-NOTE: Fill this part as a summary of your review. It is recommended to start from the "Review" section below in the template.
+In Crisis: project appears to have several outstanding, critical issues that require attention, clarity, and/or direct guidance from CNCF in order to return to a state of good health and standing.
```

Comment:
This states "CNCF". Do we need to list the various entities that would need to provide direction? Should this be explicit as to who?

Reply:
I don't think so; this is the template for reviewers, not the public document for projects.